### ISSUE TITLE
API for fetching loot tracker data (.json)

### ISSUE THREAD

**Issue by Lorentary** [runelite/runelite#10207]
Maybe this is already a feature and I'm just not aware. But here goes..

Is it possible to query the loot tracker export through a link?
I'm specifically interested in which of the two statements are true
1. The data on "https://runelite.net/account/loot-tracker" is fetched directly from the Runelite client and the loot tracker is on saved on a server. Therefore the **Export Loot Tracker** button creates a .json in the moment a download is requested

OR
 
2. The data on "https://runelite.net/account/loot-tracker" resides on a server and is displayed through decoding of a .json file - the same file which is accessed when downloading the .json

TL:DR Is there a link to the raw .json data such that the data can be accessed with a query without the need to download a file or is the .json data created upon download?

**Comment by deathbeam** [comment]
https://static.runelite.net/api/http-service/

**Comment by Lorentary** [comment]
> https://static.runelite.net/api/http-service/

Thanks for the link.. 

I however, have no idea how to use it to actually GET my loot tracker data since I "don't have access" when I try importing it into programs like excel (power query). 
It asks for a login, but the only runelite related login I have - my google account - does not work.

I'm using the link http://api.runelite.net/runelite-1.5.38/loottracker but it isn't working

**Comment by Lorentary** [comment]
You were quite quick to close this thread...

**Comment by Kamielvf** [comment]
The functionality is there, if you are having issues implementing it I suggest you join the development discussion on Discord, as it is easier to provide support that way.
